src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1499,7 +1542,12 @@ const BasicUserInfo: React.FC<{
             <p>{text}</p>
             {verifyButton}
             {cryptoEnabled && (
```

Comment:
argh again. This condition means we don't even show the `DevicesSection` when crypto is disabled.

Reply:
https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/right_panel/UserInfo.tsx#L255 If the crypto is disabled, the `DevicesSection` will display spinner. I think it's not worth it to display it (that's the behavior before this PR)

However, I suggest if we want to change deeper the behavior of the `UserInfo` to handle properly the case of disabled crypto, to make in another PR/Issue

Reply:
AFAICT `DevicesSection` isn't used at all when crypto is disabled? Line 255 is never reached.

I agree we shouldn't try and fix the whole of UserInfo. But we should try to avoid making it worse by adding misleading comments.

Reply:
Let's get rid of the new comment on line 256: https://github.com/matrix-org/matrix-react-sdk/pull/11709/files#diff-8a23c448fba779952d78ad75bb111b9ac1c2b2d923dbe2b85fce401ab9c88a9cR256

Reply:
> AFAICT `DevicesSection` isn't used at all when crypto is disabled? Line 255 is

Yes, I was talking about the case when we remove the `cryptoEnabled` condition to display `DevicesSection`.